### DISCUSSION TITLE
Welcome to typescript-reddit-clone Discussions!

### DISCUSSION THREAD

**Discussion by gianlucajahn** [gianlucajahn/typescript-reddit-clone#2] [Announcements]
## 👋 Welcome!
  I'm using Discussions as a place to connect with other members of our community. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell me about what you do with this community.

**Comment by simplesunny** [comment]
Does it have backend?
